sdks/java/io/expansion-service/build.gradle

```diff
@@ -32,10 +33,14 @@ ext.summary = "Expansion service serving several Java IOs"
 
 dependencies {
   compile project(":sdks:java:expansion-service")
+  permitUnusedDeclared project(":sdks:java:expansion-service")
```

Comment:
link to the jira (just once at the beginning is okay, doesn't have to be on every line)

Reply:
If you don't mind, I'll do it on every line. Adding it to every line allows easier identification of lines to be looked at by whoever works on BEAM-11761.